docs/src/components/Fragment.tsx

```diff
@@ -58,35 +47,28 @@ export const Fragment = ({ children, platforms }: FragmentProps) => {
       loading({ error, isLoading }) {
         if (error) {
           return (
-            <div className="p-4 rounded-md bg-yellow-50">
-              <div className="flex">
-                <div className="flex-shrink-0">
-                  <ExclamationIcon
-                    className="w-5 h-5 text-yellow-400"
-                    aria-hidden="true"
-                  />
-                </div>
-                <div className="ml-3">
-                  <p className="m-0 text-sm font-medium text-yellow-800">
-                    {error.message.includes('Cannot find module') ? (
-                      <>Content missing for {platform}.</>
-                    ) : (
-                      error.message
-                    )}{' '}
-                    <a href="https://github.com/aws-amplify/amplify-ui/issues/new/choose">
-                      Please open an issue.
-                    </a>
-                  </p>
-                </div>
-              </div>
-            </div>
+            <Alert variation="warning">
+              {error.message.includes('Cannot find module') ? (
+                <>Content missing for {platform}.</>
+              ) : (
+                error.message
+              )}{' '}
+              <Link
+                isExternal
+                href="https://github.com/aws-amplify/amplify-ui/issues/new/choose"
+              >
+                Please open an issue.
```

Comment:
Side question - What are we going to show for unsupported platforms for the primitives? Not sure this is the most user friendly message to show for say the Text Field primitive if they click the Vue button.

Reply:
Yea we should definitely update this to be more friendly and be more explicit in the docs how we talk about frameworks.

Reply:
I would like there to be a Discussion that we can link to and request customers vote & comment on that issue for support.